install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -420,7 +420,7 @@ containerRuntime:
   # - auto (automatically detect the container runtime)
   integration: none
   # -- Configure the path to the container runtime control socket.
-  # socketPath: /path/to/runtime.sock
+  socketPath: /path/to/runtime.sock
```

Comment:
This one probably needs some adjustment as well.

Reply:
probably before I can actually test it, maybe I should not change it :)